pages/api/v1/auth/data/getAllUsers.ts

```
@@ -0,0 +1,87 @@
+			error: error?.message || messages?.API_FETCH_ERROR,
+			status: Number(error?.name) || 500
+		};
+	}
+};
+
+const handler: NextApiHandler<UsersResponse | MessageType> = async (req, res) => {
+	storeApiKeyUsage(req);
+
+	const { page = 1, username } = req.body;
+
+	if (isNaN(page)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)